app/models/manageiq/providers/redhat/infra_manager/ovirt_services/strategies/v3.rb

```diff
@@ -204,6 +204,26 @@ def vm_reconfigure(vm, options = {})
       _log.info("#{log_header} Completed.")
     end
 
+    def remote_console_acquire_ticket(vm, userid, originating_server)
+      parsed_ticket = Nokogiri::XML(vm.with_provider_object(&:ticket))
```

Comment:
Hi, was the old version not working for v3?
There seem to be changes compared to the code before the split.

Reply:
It seems that we're not using the vm.provider_object anymore explicitly in our code base.

And due to the fact it carried the error with resolving the connection version, I'd like the idea of not using it if possible.